cdap-stream-clients/python/cdap_stream_client/config.py

```diff
@@ -21,16 +21,21 @@
 
 DEFAULT_HOST = u'localhost'
 DEFAULT_PORT = 10000
+DEFAULT_VERSION = u'v3'
+DEFAULT_NAMESPACE = u'default'
 DEFAULT_SSL = False
 DEFAULT_VERIFY_SSL_CERT = True
 
 
 class Config(object):
 
-    def __init__(self, host=DEFAULT_HOST, port=DEFAULT_PORT, ssl=DEFAULT_SSL,
-                 verify_ssl_cert=DEFAULT_VERIFY_SSL_CERT):
+    def __init__(self, host=DEFAULT_HOST, port=DEFAULT_PORT,
+                 version=DEFAULT_VERSION, namespace=DEFAULT_NAMESPACE,
```

Comment:
Probably better to add new arguments at the end of the existing ones to prevent accidental runtime error for existing code calling this constructor?